cypress/support/functions/index.js

```diff
@@ -9,6 +9,7 @@ const toggleNotFrequentFilters = () => {
 };
 
 const selectFrequentFilterWidget = () => {
+  cy.clickElementWithClass(`.meta-icon-calendar.input-icon-right`, true);
```

Comment:
I'm not sure this is 100% correct. do all Frequent Filter Widgets have the calendar icon? (I would expect some are used to search for bpartner or something else, which is not a date)

Reply:
Indeed I haven't checked all the frequent filters, just some of them, but the bpartner or the other names/similar criteria, I've noticed that are included in not frequent filters rather than in frequent filter. I'll have to check.

Reply:
I haven't found one which does not have a calendar icon, but I agree with you, maybe there is one. In this case, should I create a function, smthing like selectFrequentFilterWidgetWithDate() ? as I need it as it looks right now, and there are some other frequent filter widgets which also have the calendar icon, wdyt?

Reply:
We can also take the easy approach and pass a boolean parameter to the existing function `isDate`?
Fix this as you want 👍

Reply:
done.